### PR TITLE
New Package: Focusky-4.8.001

### PR DESCRIPTION
**Pull request by CoolPlayLin**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/110504&drop=dogfoodAlpha

### CODE REVIEW

**Comment by CoolPlayLin** [comment]
@Trenly

File: manifests/f/Focusky/Focusky/4.8.001/Focusky.Focusky.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+Publisher: Guangzhou Wancai Information Technology Co., Ltd
+PublisherUrl: https://www.wancaiinfo.com/
+PublisherSupportUrl: https://www.wancaiinfo.com/
+PrivacyUrl: https://www.focusky.com.cn/privacy_policy
+Author: Guangzhou Wancai Information Technology Co., Ltd
+PackageName: Focusky
+PackageUrl: https://www.focusky.com.cn/
+License: 专有软件
+Copyright: Copyright © 2023 Guangzhou Wancai Information Technology Co., Ltd
+CopyrightUrl: https://www.focusky.com.cn/
```
Comment (Trenly) [inline manifests/f/Focusky/Focusky/4.8.001/Focusky.Focusky.locale.en-US.yaml:16]:
```suggestion
CopyrightUrl: https://www.focusky.com.cn/terms-of-service
```

File: manifests/f/Focusky/Focusky/4.8.001/Focusky.Focusky.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+PackageVersion: 4.8.001
+PackageLocale: en-US
+Publisher: Guangzhou Wancai Information Technology Co., Ltd
+PublisherUrl: https://www.wancaiinfo.com/
+PublisherSupportUrl: https://www.wancaiinfo.com/
+PrivacyUrl: https://www.focusky.com.cn/privacy_policy
+Author: Guangzhou Wancai Information Technology Co., Ltd
+PackageName: Focusky
+PackageUrl: https://www.focusky.com.cn/
+License: 专有软件
```
Comment (Trenly) [inline manifests/f/Focusky/Focusky/4.8.001/Focusky.Focusky.locale.en-US.yaml:14]:
```suggestion
License: Proprietary
LicenseUrl: https://www.focusky.com.cn/user-agreement
```

File: manifests/f/Focusky/Focusky/4.8.001/Focusky.Focusky.locale.zh-CN.yaml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+Publisher: 广州万彩信息技术有限公司
+PublisherUrl: https://www.wancaiinfo.com/
+PublisherSupportUrl: https://www.wancaiinfo.com/
+PrivacyUrl: https://www.focusky.com.cn/privacy_policy
+Author: 广州万彩信息技术有限公司
+PackageName: Focusky动画演示大师
+PackageUrl: https://www.focusky.com.cn/
+License: 专有软件
+Copyright: Copyright © 2023 广州万彩信息技术有限公司版权所有
+CopyrightUrl: https://www.focusky.com.cn/
```
Comment (Trenly) [inline manifests/f/Focusky/Focusky/4.8.001/Focusky.Focusky.locale.zh-CN.yaml:16]:
```suggestion
CopyrightUrl: https://www.focusky.com.cn/terms-of-service
```

File: manifests/f/Focusky/Focusky/4.8.001/Focusky.Focusky.locale.zh-CN.yaml [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+PackageVersion: 4.8.001
+PackageLocale: zh-CN
+Publisher: 广州万彩信息技术有限公司
+PublisherUrl: https://www.wancaiinfo.com/
+PublisherSupportUrl: https://www.wancaiinfo.com/
+PrivacyUrl: https://www.focusky.com.cn/privacy_policy
+Author: 广州万彩信息技术有限公司
+PackageName: Focusky动画演示大师
+PackageUrl: https://www.focusky.com.cn/
+License: 专有软件
```
Comment (Trenly) [inline manifests/f/Focusky/Focusky/4.8.001/Focusky.Focusky.locale.zh-CN.yaml:14]:
```suggestion
License: 专有软件
LicenseUrl: https://www.focusky.com.cn/user-agreement
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by stephengillie — APPROVED** [review]

**Pull request merged**